api/src/test/java/org/apache/iceberg/AssertHelpers.java

```diff
@@ -137,4 +139,16 @@ private static void handleException(String message,
       throw e;
     }
   }
+
+  /**
+   * A convenience method to assert the cause of thrown exception.
```

Comment:
Javadoc looks wrong, cut and paste error?

Reply:
Good catch, thanks!